.github/workflows/smile-server.yaml

```diff
@@ -57,7 +57,7 @@ jobs:
 
           # calculate the version and all tags
           if [ "$BRANCH" == "main" ]; then
-            version=$(grep '"version"' www/package.json | sed 's/.*"\([0-9\.]*\)", *$/\1/')
+            VERSION=$(grep '"version"' package.json | sed 's/.*"\([0-9\.]*\)", *$/\1/')
```

Comment:
the package.json is still under the `www` folder though?

Reply:
AH i see cuz we set the working directory 
```
working-directory: www
```

Reply:
yes it was it